cmd/zed/dev/indexfile/create/command.go

```diff
@@ -113,6 +113,7 @@ func (c *Command) buildTable(zctx *zed.Context, reader zio.Reader) (*index.MemTa
 		return nil, err
 	}
 	table := index.NewMemTable(zctx, c.keys)
+	ctx := expr.NewContext()
```

Comment:
In the same way we use `zctx` as the name for a `zed.Context` to avoid confusion with `context.Context`, I think we should stick with `ectx` as the name for these things.

Reply:
Ugh that's a lot of typing